go/gregor/mem_sm.go

```diff
@@ -1,4 +1,4 @@
-package storage
+package gregor
```

Comment:
In other words, why not have `go/gregor` and `go/gregor/storage` as two different directories under `keybase/client`.  It's too late now, but it would have saved you all of the renaming...